### ISSUE TITLE
High Performance Julia 中文翻译

### ISSUE THREAD

**Issue by Roger-luo** [JuliaCN/old.juliacn.github.io#10]
应之前在群里的一些建议, 我联系了packt, 回信如下

> We appreciate the time you spent writing about your experiences with us. I would like to inform you that we offer our titles to the renowned publisher, who can translate & publish our titles in local languages.
> 
> Please let us know if you have any publishers in contact from China, who’ll be interested in this project. I’ll be glad to assist you further.

所以剩下应该是需要找一个合适的国内出版商, 这方面不知道有没有建议? 另外如果对翻译工作感兴趣也请联系我吧.

**Comment by GaZ3ll3** [comment]
你说的是 Julia High Performance 吧. 查了下,是新书, 看目录没觉得很吸引人. 如果有需求的话,我们可以自己写一本类似的开源书.

**Comment by Roger-luo** [comment]
@GaZ3ll3 是的,是前一阵才出版的新书. 群里有一些人想买. 不过我觉得没有文档好. 自己写一本类似的开源书, 在小聚的时候也有人(@sfchen)提议过. 不知道大家对出一本书有什么想法? @JuliaCN/editor

**Comment by GaZ3ll3** [comment]
good, 他的目录可以借鉴,但是看子目录就觉得不够充实了, 至少涵盖的部分不够广. 我们如果自己写, 不妨考虑下到底什么样的书更有意义,  最好不要和文档有太多重合的部分, 也不要成了一本介绍各种package的书.

**Comment by wlbksy** [comment]
这个出版社不是以出版“抄文档都抄不好”的书而出名么……
